src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/ThemeServiceTests.cs

```diff
@@ -1544,94 +2039,122 @@ await using (var context = InMemoryContentDbContext(contextId))
             {
                 await context.AddAsync(theme);
                 await context.AddRangeAsync(releaseFiles);
-
                 await context.SaveChangesAsync();
             }
 
             await using (var context = InMemoryContentDbContext(contextId))
             {
                 var service = BuildThemeService(context);
 
-                var result = await service.GetPublicationTree(PublicationTreeFilter.AnyData);
+                var result = await service.GetPublicationTree(PublicationTreeFilter.DataCatalogue);
 
                 Assert.Single(result);
                 Assert.Equal("Theme A", result[0].Title);
+                Assert.Equal("Theme A summary", result[0].Summary);
 
                 Assert.Single(result[0].Topics);
                 Assert.Equal("Topic A", result[0].Topics[0].Title);
 
                 var publications = result[0].Topics[0].Publications;
 
-                Assert.Single(publications);
+                Assert.Equal(2, publications.Count);
+                Assert.Equal("publication-a", publications[0].Slug);
                 Assert.Equal("Publication A", publications[0].Title);
+                Assert.Equal(PublicationType.NationalAndOfficial, publications[0].Type);
+                // Publication has a legacy url but it's not set because Releases exist
+                Assert.Null(publications[0].LegacyPublicationUrl);
+
+                Assert.Equal("publication-b", publications[1].Slug);
+                Assert.Equal("Publication B", publications[1].Title);
+                Assert.Equal(PublicationType.NationalAndOfficial, publications[1].Type);
+                Assert.Null(publications[1].LegacyPublicationUrl);
             }
         }
 
         [Fact]
-        public async Task GetPublicationTree_AnyData_DoesNotFilterPublicationWithNoDataOnLatestRelease()
+        public async Task GetPublicationTree_DataCatalogue_MultipleThemesTopics()
```

Comment:
These test cases are a little inconsistent as we don't seem to have the `MultipleThemesTopics` on some of the other filter types. We might want to add this case for all filter types.

Reply:
There is one for `FindStatistics`, `DataTables`, and `DataCatalogue`, which covers everything - I've renamed the `FindStatistics` one from `GetPublicationTree_MultipleThemesTopics` to `GetPublicationTree_FindStatistics_MultipleThemesTopics` so it's clearer